syntax = "proto3";

message User {
    int32 id = 1;
    string name = 2;
    string email = 3;
}

message GetUserRequest {
    int32 id = 1;
}

message UserResponse {
    repeated User users = 1;
}

service UserService {
    rpc GetUser (GetUserRequest) returns (UserResponse);
}
